### ISSUE TITLE
Some issues about reproducing the baseline

### ISSUE THREAD

**Issue by wei0699** [muhanzhang/SEAL#43]
Hi, Dr. Zhang
The code you provided is very detailed so I can learn a lot, thank you! However, I still have some problems when reproducing the benchmarks. So, I hope you can give me some guidance. 
The first problem, I reproduced the node2vec on "facebook" without changing anything, but the AUC is just about 0.5(in your paper, it is 0.99 ), I have tried many times and used diffferent "p" and "q", but the result had no improve, is there any detail I haven’t noticed? Meanwhile, the embedding method "SPC" could produce the some result as in the paper. 
The second problem, when I used the dataset "arxiv", the code "cmd = sprintf('python ../../software/node2vec/src/main.py --input %s.edgelist --output %s.emd --p %f --q %f --dimensions 128 --window-size 10', data_name, data_name, p, q); system(cmd);"  could run successfully but the result "data_name.emd" could not be saved in tempdata, but in other datasets, this problem could not happen,  so what could be the reason?
By the way, I reproduced these experiments under windows10, using matlab2017a.
Thank you very much!

**Comment by muhanzhang** [comment]
I only have some guessings for your problems. 1) Can you try to manually check whether the node2vec embeddings are successfully generated and loaded? An AUC of 0.5 indicates random, which either means the embeddings are not trained at all or there is something wrong with loading them. Also, does node2vec work on other datasets? 2) What's the error message about not saving in tempdata? Is it because you are out of disk space?

**Comment by wei0699** [comment]
> Can you try to manually check whether the node2vec embeddings are successfully generated and loaded? An AUC of 0.5 indicates random, which either means the embeddings are not trained at all or there is something wrong with loading them. Also, does node2vec work on other datasets? 2) What's the error message about not saving in tempdata? Is it because you are out of disk space?
Thanks for your repley. 1) I tested the commend 'python ../../software/node2vec/src/main.py --input %s.edgelist --output %s.emd --p %f --q %f --dimensions 128 --window-size 10', data_name, data_name, p, q' in CMD(win10), it could generate and save "facebook_5.emd" in folder "data/embedding", but the AUC result was bad, this problem also existed in other datesets like "USAir" and "wikipedia". I also found that the absolute value of each element in embeddings is around[0.0001, 0.001], I'm not sure if this means the learned embedding is invalid. 2) The error meddage is "No such file or director" and I also test the commend in CMD with the input "arxiv_5.edgelist", it can run successfully but no output.

**Comment by muhanzhang** [comment]
I see.  Can you try to manually make a directory "tempdata/" and see if the problem exists?

**Comment by wei0699** [comment]
Thanks for your suggestions and I have solved this problem!  It is indeed caused by the path problem but this problem exists in the original "node2vec" code. After I modified the default save path of input and output in "node2vec/src/main.py", the above two problems were solved at the same time. However, at present, I don’t know why this problem only exists when running some of the datasets.  But, it doesn't metter, I have been able to successfully complete the experiment, thanks for your code and help, I have benefited a lot!
